src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -403,7 +567,7 @@ unittest
 
 ******************************************************************************/
 
-public long parseTimeInterval ( cstring value )
+private long parseTimeInterval ( cstring value )
```

Comment:
This is another breaking change in the API that needs to be addressed.

Reply:
reverted to public